src/main/java/duke/initials/Deadline.java

```diff
@@ -0,0 +1,25 @@
+package duke.initials;
+
+public class Deadline extends Task {
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
```

Comment:
could name this as date instead to make to clearer

Reply:
named it as by because of the differentiation between Deadline and Event of by and at. It helped me differentiate clearer on my end